src/libecalc/presentation/yaml/mappers/variables_mapper/variables_mapper.py

```diff
@@ -112,7 +112,8 @@ def map_yaml_to_variables(
     result_options: dto.ResultOptions,
 ) -> dto.VariablesMap:
     timeseries_collections = [
-        TimeSeriesCollectionMapper(resources).from_yaml_to_dto(timeseries) for timeseries in configuration.time_series
+        TimeSeriesCollectionMapper(resources).from_yaml_to_dto(timeseries.dict(by_alias=True))
```

Comment:
We should look into removing this mapper and the classes, should be able to handle it in the yaml classes?

Converting back to dict as that is expected in TimeSeriesCollectionMapper

Reply:
you mean to implicitly and automatically handle the conversion?

Reply:
https://equinor-ecalc.atlassian.net/browse/ECALC-586 Explained a bit here. 
TimeSeriesCollectionMapper should not need to deal with validation of the yaml at least, it could be simplified to only deal with reading the resources and creating something that can easily be added to variables map